readthedocs/builds/constants.py

```diff
@@ -51,6 +51,12 @@
     (EXTERNAL, EXTERNAL_TEXT),
     (UNKNOWN, UNKNOWN_TEXT),
 )
+EXTERNAL_VERSION_STATE_OPEN = "open"
+EXTERNAL_VERSION_STATE_CLOSED = "closed"
```

Comment:
Do we want more granularity here? `Merged` vs `Closed`?

Reply:
In the first commit, I used `open`, `closed` and `merged`, but I didn't find how/where we would use it differently, so I removed it because it just complicate the logic. Do you have a good usage for this in mind?

Reply:
Mostly displaying the information in the build page, or docs pages. Not a huge deal though, seems fine to start with this.